python/vyos/nat.py

```diff
@@ -17,10 +17,30 @@
 from vyos.template import bracketize_ipv6
 
 
+def log_nat_type_str(nat_type):
+    nat_types = {
+        'destination': 'DST',
+        'source': 'SRC',
+        'output': 'OUT',
+    }
+
+    return nat_types.get(nat_type, 'SRC')
+
+
+def log_translation_prefix(nat_type):
+    translation_prefixes = {
+        'destination': 'd',
+        'source': 's',
+        'output': 'd',
+    }
+
+    return translation_prefixes.get(nat_type, 'd')
+
+
 def parse_nat_rule(rule_conf, rule_id, nat_type, ipv6=False):
     output = []
     ip_prefix = 'ip6' if ipv6 else 'ip'
-    log_prefix = ('DST' if nat_type == 'destination' else 'SRC') + f'-NAT-{rule_id}'
+    log_prefix = log_nat_type_str(nat_type) + f'-NAT-{rule_id}'
```

Comment:
Is there a good reason to mix string concatenation with an f-string? Can we use a simple dict of types and use something like:

```suggestion
    log_prefix = log_nat_type_str(f'{nat_types["nat_type"]}-NAT-{rule_id}'
```

`dict.get()` will make the script blow up at the concatenation if a nonexistent NAT type is requested. I believe it's better if it blows up at the dictionary lookup stage because the exception trace will include the offending key.

Reply:
I used func() + f-str because I only replaced the if block,

Do you mean I make nat_types `global` in this file and use simple `f'{nat_types["nat_type"]}-NAT-{rule_id}'` for this?